.github/workflows/release.yml

```diff
@@ -9,6 +9,7 @@ on:
   push:
     branches:
       - main
+      - eternal_ci
```

Comment:
This is solely for the purpose of demonstrating this PR works.

Reply:
That's what we've got the `workflow_dispatch` trigger for. You can always manually run this workflow on any branch [from the actions tab](https://github.com/phylum-dev/cli/actions/workflows/release.yml)